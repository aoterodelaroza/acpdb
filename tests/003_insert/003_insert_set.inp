## check: 003_insert_set.out -a1e-10
## delete: 003_insert_set.db
## labels: regression quick

verbose
system rm -f 003_insert_set.db
connect 003_insert_set.db

insert litref bibtex ../zz_source/bsip2.bib
insert set s22x5
 description A description.
 litrefs bsip1 boys1970
end
