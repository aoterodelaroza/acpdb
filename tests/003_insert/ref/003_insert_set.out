* SYSTEM: rm -f 003_insert_set.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 003_insert_set.db
Creating skeleton database 

* INSERT: insert data into the database (LITREF)

* INSERT: insert data into the database (SET)

